web/react/components/rename_team_modal.jsx

```diff
@@ -54,37 +57,36 @@ module.exports = React.createClass({
         $(this.refs.modal.getDOMNode()).off('hidden.bs.modal', this.handleClose);
     },
     getInitialState: function() {
-        return { name: this.props.teamDisplayName };
+        return {name: this.props.teamDisplayName};
     },
     render: function() {
-
-        var name_error = this.state.name_error ? <label className='control-label'>{ this.state.name_error }</label> : null;
-        var server_error = this.state.server_error ? <div className='form-group has-error'><label className='control-label'>{ this.state.server_error }</label></div> : null;
+        var name_error = this.state.name_error ? <label className='control-label'>{this.state.name_error}</label> : null;
+        var server_error = this.state.server_error ? <div className='form-group has-error'><label className='control-label'>{this.state.server_error}</label></div> : null;
 
         return (
-            <div className="modal fade" ref="modal" id="rename_team_link" tabIndex="-1" role="dialog" aria-hidden="true">
-                <div className="modal-dialog">
-                    <div className="modal-content">
-                        <div className="modal-header">
-                            <button type="button" className="close" data-dismiss="modal">
-                                <span aria-hidden="true">&times;</span>
-                                <span className="sr-only">Close</span>
+            <div className='modal fade' ref='modal' id='rename_team_link' tabIndex='-1' role='dialog' aria-hidden='true'>
+                <div className='modal-dialog'>
+                    <div className='modal-content'>
+                        <div className='modal-header'>
+                            <button type='button' className='close' data-dismiss='modal'>
+                                <span aria-hidden='true'>&times;</span>
+                                <span className='sr-only'>Close</span>
```

Comment:
@nickago If there's one button on the bottom right, it should be "Close", if there are two buttons, one should be "Cancel" (on left) the other should be the action button, e.g. "Save", "Rename Team", etc.